pkg/api/types.go

```diff
@@ -1205,6 +1221,13 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// Local represents a directory with specific topology contraints
+type LocalVolumeSource struct {
+	// The full path to the volume on the node
+	// For alpha, this path must be a file-based volume
+	Path string
```

Comment:
The main argument for overloading is to be consistent with the other volume types.  For example, GCE PD does not distinguish between if the disk has already been formatted or not.  I agree that for block case, we need to add fsType to the structure, similar to how gcepd does it.  I wanted to leave that out for now until we have block support.